src/linux/backend-v4l2.cpp

```diff
@@ -1913,6 +1928,12 @@ namespace librealsense
             }
         }
 
+        bool v4l_uvc_device::video_is_mplane()
+        {
+            return _dev.type == V4L2_BUF_TYPE_VIDEO_CAPTURE_MPLANE ||
```

Comment:
Here we count MPLANE as CAPTURE + OUTPUT but everywhere else we only question CAPTURE.
What will happen in `OUTPUT` buffers?
we only allow this right?
`if(!(cap.capabilities & (V4L2_CAP_VIDEO_CAPTURE_MPLANE | V4L2_CAP_VIDEO_CAPTURE)))`

Reply:
Better to check what we really support..